src/cmr/sizing/core.clj

```diff
@@ -22,32 +22,56 @@
 ;;; Note that all internal operations for estimating size are performed
 ;;; assuming bytes as the units.
 
-(defn get-measurement
+(defn- get-measurement
   [variable]
   (let [dims (get-in variable [:umm :Dimensions])
         total-dimensionality (reduce * (map :Size dims))
         data-type (get-in variable [:umm :DataType])]
     (* total-dimensionality
        (util/data-type->bytes data-type))))
 
+(defn- get-avg-gran-size
+  [vars]
+  (let [value (get-in (first vars) [:umm :SizeEstimation :AvgCompressionRateASCII])]
```

Comment:
Looks like the functions for size and compression have the values swapped.

Reply:
Whoops